src/gluonts/model/forecast.py

```diff
@@ -442,6 +442,7 @@ def __init__(
         self,
         samples: np.ndarray,
         start_date: pd.Period,
+        quantiles: Optional[Dict] = None,
```

Comment:
We have a dedicated classed `QuantileForecast`. Do we want to support cases where we have both samples and quantiles?

/cc @lostella